tests/ComputeSharp.Tests/Attributes/AdditionalResourceAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ComputeSharp.Tests.Attributes;
+
+/// <summary>
+/// An attribute to use with <see cref="CombinatorialAttribute"/> targeting an additional specific resource type.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+public sealed class AdditionalResourceAttribute : Attribute
```

Comment:
What does this do? I'm a bit confused on what this sets up.

Reply:
Ahah this is a magic attribute for my `[CombinatorialTestMethod]` setup.
You can see it being used here:

https://github.com/Sergio0694/ComputeSharp/blob/da90dd9157aaa07ca6834a50380e65ce1914a0f3/tests/ComputeSharp.Tests/BufferTests.cs#L227-L240

What this does is it adds another combinatorial sequence of optional arguments to a given test method. For instance, in this case this method is actually being invoked a total of `[DEVICES]x[RESOURCES]x[ADDITIONAL_RESOURCES]x[DATA]` times. That is, this single method gets invoked a total of 90 times with all possible combinations of parameters specified here, without the need to write all those tests manually 😄